doc/1/controllers/document/m-update/index.md

```diff
@@ -0,0 +1,156 @@
+
+The `mUpdate` method takes into account those following arguments:
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `retryOnConflict`  | <pre>Integer</pre> (optional)                | The number of times the database layer should retry in case of version conflict |
+| `waitForRefresh`   | <pre>Boolean</pre> (optional)                | If set to `true`, Kuzzle will wait for the persistence layer to finish indexing |
+
+
+## Return
+
+A `ConcurrentHashMap<String, ArrayList<Object>>` which has a `successes` and `errors` `ArrayList<Object>`:
+Each created document is an object of the `successes` array with the following properties:
+
+| Property     | Type                                         | Description                      |
+|------------- |--------------------------------------------- |--------------------------------- |
+| `_source`    | <pre>ConcurrentHashMap<String, Object></pre> | Updated document                 |
+| `_id`        | <pre>String</pre>                            | ID of the updated document       |
+| `_version`   | <pre>Integer</pre>                           | Version of the document in the persistent data storage |
+
+Each errored document is an object of the `errors` array with the following properties:
+
+| Property     | Type                                         | Description                      |
+|------------- |--------------------------------------------- |--------------------------------- |
+| `document`   | <pre>ConcurrentHashMap<String, Object></pre> | Document that causes the error   |
+| `status`     | <pre>Integer</pre>                           | HTTP error status                |
+| `reason`     | <pre>String</pre>                            | Human readable reason |
+
+## Usage
+
+<<< ./snippets/m-update-java.java
+
+
+:::
+::: tab Kotlin
+
+## Arguments
+
+```kotlin
+  fun mUpdate(
+      index: String,
+      collection: String,
+      documents: ArrayList<ConcurrentHashMap<String, Any?>?>,
+      waitForRefresh: Boolean? = null,
+      retryOnConflict: Int? = null): CompletableFuture<ConcurrentHashMap<String, ArrayList<Any>?>>
+```
+
+| Arguments          | Type                                                    | Description                       |
+| ------------------ | ------------------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                                       | Index                             |
+| `collection`       | <pre>String</pre>                                       | Collection                        |
+| `documents`        | <pre>ArrayList<ConcurrentHashMap<String, Any?>></pre> | ArrayList containing the documents to update |
+| `options`          | <pre>UpdateOptions</pre><br>(`null`)                    | Query options |
```

Comment:
```suggestion
| `retryOnConflict`  | <pre>Int</pre> (optional)                | The number of times the database layer should retry in case of version conflict |
| `waitForRefresh`   | <pre>Boolean</pre> (optional)                | If set to `true`, Kuzzle will wait for the persistence layer to finish indexing |
```